docs/run-core-node/configuring.mdx

```diff
@@ -70,7 +70,7 @@ Configuring a node to participate in SCP and sign messages is a three step proce
 - Add `NODE_SEED="SD7DN..."` to your configuration file, where `SD7DN...` is the secret key from the keypair
 - Add `NODE_IS_VALIDATOR=true` to your configuration file
 
-If you want other validators to add your node to their quorum sets, you should also share your public key (GDMTUTQ... ) by publishing a stellar.toml file on your homedomain following specs laid out in [SEP-20](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0020.md).
+If you want other validators to add your node to their quorum sets, you should also share your public key (GDMTUTQ... ) by publishing a stellar.toml file on your home domain following specs laid out in [SEP-20](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0020.md).
```

Comment:
homedomain is the name of the option in the sdk and the name of the data entry.

Reply:
I would remove this suggestion then

Reply:
That same sections refers to an environment variable `NODE_HOME_DOMAIN`; to my ear "home domain" make more sense in this context.  However it is mentioned as "homedomain" in [SEP-20](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0020.md) so I have now reverted it here.